docs/csharp/language-reference/configure-language-version.md

```diff
@@ -1,64 +1,69 @@
+If you must specify your C# version explicitly, you can do so in several ways:
 
-![Screenshot of advanced build settings where you can specify the language version](./media/configure-language-version/advanced-build-settings.png)
-
-> [!NOTE]
-> If you use the Visual Studio IDE to update your csproj files, the IDE
-> creates separate nodes for each build configuration. You'll typically
-> set the value the same in all build configurations, but you need to
-> set it explicitly for each build configuration, or select "All Configurations"
-> when you modify this setting. You'll see the following in your csproj file:
->
->```xml
-> <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
->  <LangVersion>latest</LangVersion>
-></PropertyGroup>
->
-> <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
->  <LangVersion>latest</LangVersion>
-> </PropertyGroup>
-> ```
->
+- Manually edit your [project file](#edit-the-project-file).
+- Set the language version [for multiple projects in a subdirectory](#configure-multiple-projects).
+- Configure the [`-langversion` compiler option](#set-the-langversion-compiler-option).
 
-## Edit the csproj file
+### Edit the project file
 
-You can set the language version in your **.csproj** file. Add an element like the following:
+You can set the language version in your project file. For example, if you explicitly wanted access to preview features, you could do add an element like this:
 
 ```xml
 <PropertyGroup>
-   <LangVersion>latest</LangVersion>
+   <LangVersion>preview</LangVersion>
 </PropertyGroup>
 ```
 
-The value `latest` uses the latest minor version of the C# language. Valid values are:
+The value `preview` uses the latest available preview C# language that your compiler supports.
+
+### Configure multiple projects
+
+You can create a **Directory.Build.props** file that contains the `<LangVersion>` element to configure multiple directories. You typically do that in your solution directory. Add the following to a **Directory.Build.props** file in your solution directory:
+
+```xml
+<Project>
+ <PropertyGroup>
+   <LangVersion>preview</LangVersion>
+ </PropertyGroup>
+</Project>
+```
+
+Now, builds in every subdirectory of the directory containing that file will use C# version 7.3 syntax. For more information, see the article on [Customize your build](/visualstudio/msbuild/customize-your-build).
+
+### Set the langversion compiler option
+
+You can use the `-langversion` command-line option. For more information, see the article on the [-langversion](../language-reference/compiler-options/langversion-compiler-option.md) compiler option. You can see a list of the valid values by typing  `csc -langversion:?`.
```

Comment:
Hmm. `csc` won’t be available on a Mac or Linux system with only the .NET Core SDK installed. Should we have an equivalent `dotnet build` option?

Reply:
I wonder if it's worth having this section given this, actually. Generally we'll be messaging LangVersion.

Reply:
@cartermp I think that’s the best resolution: I’d delete this paragraph.